content/zh-cn/docs/concepts/workloads/pods/disruptions.md

```diff
@@ -282,14 +285,13 @@ hornoring the
 中的 `terminationGracePeriodSeconds` 配置值。
 
 <!--
-## PDB Example
-
+## PodDisruptionBudget example {#pdb-example}
 Consider a cluster with 3 nodes, `node-1` through `node-3`.
 The cluster is running several applications.  One of them has 3 replicas initially called
 `pod-a`, `pod-b`, and `pod-c`.  Another, unrelated pod without a PDB, called `pod-x`, is also shown.
 Initially, the pods are laid out as follows:
 -->
-## PDB 例子   {#pdb-example}
+## PodDisruptionBudget（PDB） 例子   {#pdb-example}
```

Comment:
```suggestion
## PodDisruptionBudget 例子   {#pdb-example}
```
As a heading or title, it is few to use the form of `PodDisruptionBudget (PDB)`, which often appears in the begining of body.